src/utils/collapseSourcemaps.ts

```diff
@@ -49,23 +48,20 @@ class Link {
 		const sourcesContent: string[] = [];
 		const names: string[] = [];
 
-		const mappings = this.mappings.map(line => {
+		const mappings = [];
```

Comment:
It may help performance to initialize the array length there? `new Array(this.mappings.length)`

Reply:
No, V8 is very good at growing arrays. Besides, `new Array(length)` creates a sparse array (all items are undefined), which is a perf antipattern.

Reply:
Ah right ok.